ecosante/newsletter/models.py

```diff
@@ -673,8 +705,11 @@ class NewsletterDB(db.Model, Newsletter):
     allergenes: dict = db.Column(postgresql.JSONB)
     raep_debut_validite = db.Column(db.String())
     raep_fin_validite = db.Column(db.String())
+    indice_uv_label: str = field(default=None, init=True)
+    indice_uv_value: int = field(default=None, init=True)
```

Comment:
Deux choses ici, le `field` c’est pour les `dataclass` python, pas pour un objet SQLAlchemy, aussi je trouve ça pratique de garder l’objet indice_vu qu’on a utilisé pour le debug.
```suggestion
    indice_uv_label: str = db.Column(db.String())
    indice_uv_value: int = db.Column(db.String())
    indice_uv_id: int =db.Column(db.Integer(), db.ForeignKey(IndiceUv.id))
    indice_uv: IndiceUv = db.relationship(IndiceUv, foreign_keys=[indice_uv_id])
```

Reply:
Erreur de ma part, j'avais pas stocké en base au début, c'est un résidu